ibm/service/satellite/resource_ibm_satellite_cluster_test.go

```diff
@@ -229,8 +229,9 @@ func testAccCheckSatelliteClusterCreate(clusterName, locationName, managed_from,
 		name                   = "%s"  
 		location               = ibm_satellite_host.assign_host.0.location
 		enable_config_admin    = true
-		kube_version           = "4.9_openshift"
+		kube_version 		   = "4.11_openshift"
 		operating_system       = "%s"
+		infrastructure_topology = "single-replica"
```

Comment:
Instead of updating existing test case . Can we get a new testcase for singlereplica.
And share all testcases  releated to satellite cluster execution results as part of PR